src/@types/Watchlist.tsx

```diff
@@ -0,0 +1,12 @@
+export type Watchlist = {
+  name: string
+  description: string
+  id: string
+  projects: number
```

Comment:
Why is `projects` a number? Shouldn't it be a list of IDs (which are strings)

Reply:
This here is only used for displaying the number of projects on the page summary.
When we display the projects on the details page projects will need to be paginated so we will need a separate endpoint to handle the projects as part of watchlist.

Thus we don't need to have the id of the projects.

Reply:
In that case you should rename the the variable to make it clear.